doc/cli_tx_create.md

```diff
@@ -0,0 +1,33 @@
+# TX creator
+
+Creates a signed transaction
+
+## Building
+
+```
+cargo build --bin tx_create
```

Comment:
`transaction` would be a more fitted name. We will use this to:

* build transaction;
* decode/debug transactions (display the content and all)

Reply:
Don't we want another binary for decoding transaction in UNIX spirit? Docoding will be a separate subcommand with no common arguments which will act like a completely independent app anyway.

Reply:
that's more or less the spirit. But let be consistent too.

* we need one executable for the REST API glue (query/post);
* we need one for the address (already present) (create/debug);
* we need one for the transaction (this one) (create/sign/debug);
* we need one for the certificate (create/debug)